frontend/src/app/planner/[slug]/page.tsx

```diff
@@ -0,0 +1,49 @@
+import { Breadcrumbs, Chip, Link, Typography } from '@mui/joy'
+
+import { makeGetProject } from '@/infra/project/getProject'
+import { serverFetch } from '@/infra/lib/serverFetch'
+import { Planner } from './components/Planner'
+import { makeGetProjectStats } from '@/infra/project/getProjectStats'
+
+const getPageData = async (id: string) => {
+  const apiClient = await serverFetch()
+  const getProject = makeGetProject(apiClient)
+  const getProjectStats = makeGetProjectStats(apiClient)
+  return await Promise.all([getProject(id), getProjectStats(id)])
+}
+
+export default async function Page({ params }: { params: { slug: string } }) {
+  const [project, projectStats] = await getPageData(params.slug)
+
+  return (
+    <>
+      <Breadcrumbs aria-label="breadcrumbs">
+        <Link color="neutral" href="#">
+          Projects
+        </Link>
+        <Typography
+          level="body-lg"
+          endDecorator={
+            <Chip component="span" size="sm" sx={{ textDecoration: 'underline' }}>
```

Comment:
When I tab through the page, I notice that the link in the chip doesn't show any focus indication, which we would want for a11y purposes. The documentation does make note of how to make a chip clickable and I this this would solve the focus indication: https://mui.com/joy-ui/react-chip/#clickable